data_pipelines_cli/cli_commands/compile.py

```diff
@@ -137,10 +140,17 @@ def compile_project(
     default=False,
     help="Whether to build a Docker image",
 )
+@click.option(
+    "--save-home",
```

Comment:
This was meant to be a convenient flag from UX's POV; so at the same time one is doing `compile`, one can also add `--save-home` flag to copy `profiles.yml` to his home directory, instead of doing it in a separate step (by calling `dp gen-home-profiles`).
We can drop it, if it makes `dp compile` unnecessarily confusing.

On the other hand, do we want to have a different command like `dp prepare-vs-code`? If so, then maybe we should remove `gen-home-profiles` and move its inners to this `prepare-vs-code` command?

Reply:
ok, let's keep --save-home flag but let's change it to --save-profile-home (or something similar) because during compilation there is much more files generated.

Reply:
but there is one small problem the profile generated to home dir can't have any var or env_var - it must be replaced for the user / env.

Reply:
I've removed `--save-home` flag, as requested. And added reading vars and env_vars.